docs/pages/components/Panel.mdx

```diff
@@ -0,0 +1,261 @@
+
+Inspect the below example and you will see that the `Panel` component has an `aria-labelledby` attribute with the 
+value of the `id` of the heading.
+
+```jsx example
+<Panel heading={{ id: 'panel-heading-prop-example', text: 'Panel Heading', level: 3 }}>
+  <p>Panel Content</p>
+</Panel>
+```
+
+### Panel (Composed)
+
+In addition to using the `heading` prop, you can also compose the `Panel` component with the `PanelHeader` and 
+`PanelContent` components.
+
+<Note>
+  The `PanelHeader` and `PanelContent` should not be used with the `Panel`
+  heading prop at the same time. Choose one or the other to use. 
+  
+  In addition, if you use the `PanelHeader`, you are responsible for adding `aria-labelledby` to
+  the `Panel` and an ID to the heading.
+</Note>
+
+```jsx example
+<Panel>
+  <PanelHeader>
+    <h3>Panel Header</h3>
+  </PanelHeader>
+  <PanelContent>
+    <p>Panel Content</p>
+  </PanelContent>
+</Panel>
+```
+
+### Disabling Panel Padding
+
+For greater control over the styling of the `Panel`'s content, you can disable the default padding by setting 
+the `padding` prop to `false`.
+
+```jsx example
+<Panel
+  heading={{ id: 'example', text: 'Panel Heading', level: 3 }}
+  padding={false}
+>
+  <p style={{ paddingLeft: '1rem' }}>Panel Content</p>
+</Panel>
+```
+
+### Disabling Panel Content Padding
+
+In addition to being able to disable the `Panel` padding via its `padding` prop, the `PanelContent` also has a `padding`
+prop which can be set to `false`. An example where this may be useful is when trying to display content in a two 
+column or grid layout. 
+
+In the following example, we are using CSS `columns` to create the two column layout. 
+
+```jsx example
+<Panel>
+  <PanelHeader
+    style={{
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+      flexWrap: 'wrap'
+    }}
+  >
+    <h3>Panel Header</h3>
+    <Button variant="primary" thin>
+      <Icon type="plus" />
+      Button
+    </Button>
+  </PanelHeader>
+  <PanelContent
+    padding={false}
+    style={{
+      columns: '2',
+      columnRule: 'var(--panel-divider-border)',
+      columnGap: '0'
+    }}
+  >
+    <div style={{ padding: 'var(--space-small)' }}>Column 1</div>
+    <div style={{ padding: 'var(--space-small)' }}>Column 2</div>
+  </PanelContent>
+  <PanelContent>
+    <p>Footer</p>
+  </PanelContent>
+</Panel>
+```
+
+### Header With Flexbox Content
+
+A common use-case for the `Panel` is to display a header with a button or other content. To achieve this, you 
+can compose the `Panel` using a `PanelHeader` with a flexbox layout.
+
+```jsx example
+<Panel>
+  <PanelHeader
+    style={{
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+      flexWrap: 'wrap'
+    }}
+  >
+    <h3>Panel Header</h3>
+    <Button variant="primary" thin>
+      <Icon type="plus" />
+      Button
+    </Button>
```

Comment:
Not sure this is needed here for this example.

Reply:
Only reason I was thinking it might be a good example to show is because it follows more or less the pattern in the [Cauldron Pattern Library](https://preview.uxpin.com/520e80315786a67ba43a3ca50598e6d69782627c#/pages/161871161/simulate/no-panels?mode=cvidm). Was also thinking that we have a lot of designs that use similar patterns (e.g., heading with button/justified with space between). 

I am cool with it either way, just thought people may find it useful/save them some time!